proto/folio/fsv2/group_booking_folio_state.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package holms.types.folio.fsv2;
+option csharp_namespace = "HOLMS.Types.Folio.FSv2";
+
+import "folio/fsv2/folio_atomic_charge.proto";
+import "folio/fsv2/folio_card_refund.proto";
+import "folio/fsv2/folio_card_sale.proto";
+import "folio/fsv2/folio_check_cash_payment.proto";
+import "folio/fsv2/folio_tax_assessment.proto";
+import "primitive/monetary_amount.proto";
+import "folio/fsv2/folio_indicator.proto";
+
+message GroupBookingFolioState {
+	repeated .holms.types.folio.fsv2.FolioAtomicCharge effective_charges = 1;
+	repeated .holms.types.folio.fsv2.FolioCardRefund card_refunds = 2;
+	repeated .holms.types.folio.fsv2.FolioCardSale card_sales = 3;
+	repeated .holms.types.folio.fsv2.FolioCheckCashPayment check_cash_payments = 4;
+
+	repeated .holms.types.folio.fsv2.FolioTaxAssessment current_tax_assessments = 5;
+	repeated .holms.types.folio.fsv2.FolioTaxAssessment anticipated_ending_tax_assessment = 6;
+
+	repeated .holms.types.folio.fsv2.FolioAtomicCharge native_charges = 7;
+	repeated .holms.types.folio.fsv2.FolioAtomicCharge imported_from_other_folio_charges = 8;
+
+	.holms.types.primitive.MonetaryAmount current_net_charges = 9;
```

Comment:
Can you get rid of the word "net" here? 

This is a holdover from when we had the concept of an "adjustment" separate from a charge. We don't have this anymore, so it should be just "current_charges" and "anticipated_charges", that's all. Go ahead and make this change in the other folio state, too pls.